static/app/components/events/searchBarFields.tsx

```diff
@@ -0,0 +1,50 @@
+import omit from 'lodash/omit';
```

Comment:
If this file is exporting only constants, can we label it a `searchBarFieldConstants.tsx`?

we don't do `.ts` vs. `.tsx` files to distinguish btw component vs. helpers do we? 😅

Reply:
hey that is a good feedback - I believe we started to use only `.ts` when we do not use `jsx`. I will update and rename the file 🙂